src/Resources/contao/widgets/Password.php

```diff
@@ -114,9 +114,12 @@ protected function validator($varInput)
 			return '*****';
 		}
 
-		if (Utf8::strlen($varInput) < \Config::get('minPasswordLength'))
+		// Check password length either from DCA or use Config as fallback (#1086)
+		$intLength = ($this->minlength > 0) ? $this->minlength : \Config::get('minPasswordLength');
```

Comment:
This can be simplified to `$intLength = $this->minlength ?: \Config::get('minPasswordLength');`